constant.go

```diff
@@ -17,6 +18,8 @@ const (
 
 	// BridgeConfigType is the type of the bridge configuration
 	bridgeConfigType = "BridgeConfig"
+	// PinataIPFSStorage is the identifier of the pinata IPFS storage
+	PinataIPFSStorage = "pinata"
```

Comment:
Let this constant be above where we have the other providers....also, the name should be 

```go
// PinataCloud is the identifier of the pinata cloud storage
PinataCloud = "pinata"
```